data/extensions.yml

```diff
@@ -881,3 +881,11 @@
     - original
   tags:
     - Tooling
+-
+  name: middleman-firebase
+  description: Rake tasks for deploying Middleman sites to Firebase Hosting
+  links:
+    github: https://bitbucket.org/marnen/middleman-firebase
```

Comment:
Note that this is hosted at Bitbucket, but I can't seem to have an obvious link to the repo unless I call it `github` (and display it with a GitHub icon). Perhaps this should be more generic (particularly with some people hesitant about GitHub due to the MS acquisition...).

Reply:
@marnen Makes sense to make this more generic. Would love to see that PR

Reply:
I’ll try to get to that.